dcos_installer/upgrade.py

```diff
@@ -71,6 +71,25 @@
     exit 1
 fi
 
+# Check if the node has node/cluster checks and run them
+if 3dt check; then
+   # command exists
+   if 3dt check node-poststart; then
+      echo "Node checks passed, proceeding with upgrade"
+   else
+      echo "Cannot proceed with upgrade, node checks failed"
+      exit 1
```

Comment:
We should print runner output on failure, otherwise failures will be a pain to debug.